BRANCHES.md

```diff
@@ -16,6 +16,9 @@ API compatibility in the `master` branch between major version changes. We
 also maintain ABI compatibility within LTS branches; see the next section for
 details.
 
+New LTS branches are usually created around the time of a new major version.
```

Comment:
This doesn't match our historical practice: we've also created LTS branches at other times. I don't think the reasons for us to make LTS branches have really changed.
```suggestion
Every major version will become an LTS branch when the next major version is
released. We may occasionally create LTS branches from other releases at our
discretion.
```